src/styles/container.scss

```diff
@@ -0,0 +1,31 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  padding: 50px 40px;
```

Comment:
According to the Code Style Rules, it's recommended to use variables for often used values. Consider defining a variable for the padding value `50px 40px` in your `variables.scss` file and use it here.